src/main/java/com/casper/sdk/model/clvalue/CLValueAny.java

```diff
@@ -76,7 +80,7 @@ public void deserialize(DeserializerBuffer deser) throws ValueDeserializationExc
         try (ByteArrayInputStream bis = new ByteArrayInputStream(deser.readByteArray(objectByteArrayLength));
              ObjectInputStream ois = new ObjectInputStream(bis)) {
             this.setValue(ois.readObject());
-        } catch (IOException | ClassNotFoundException e) {
+        } catch (IOException | ClassNotFoundException | ValueSerializationException e) {
```

Comment:
I'd have probably moved the exception handling up to  abstract AbstractCLValue deserialize method and made it a concrete method, then have a new abstract method that is called in that. Then you'd only have to do it in one place rather than every class that extends AbstractCLValue something like this:

```
class AbstractCLValue {
    public abstract void deserializeCustom(DeserializerBuffer deser);
    
    public void deserialize(DeserializerBuffer deser) throws ValueDeserializationException {
        try {
            deserializeCustom(deser);
        } catch (Exception e) {
            throw new ValueDeserializationException(String.format("Error deserializing %s", this.getClass().getSimpleName()), e);
        }
    }
}
```